assets/js/googlesitekit/datastore/user/user-input-settings.js

```diff
@@ -196,16 +196,16 @@ export const baseControls = {
 	[ GET_CACHED_USER_INPUT_SETTINGS ]: () => {
 		return getItem( CACHE_KEY_NAME );
 	},
-	[ SET_CACHED_USER_INPUT_SETTING ]: createRegistryControl(
-		( registry ) =>
-			async ( { payload: { settingID, values } } ) => {
-				const settings =
-					registry.select( CORE_USER ).getUserInputSettings() || {};
+	[ SET_CACHED_USER_INPUT_SETTING ]: createRegistryControl( ( registry ) =>
+		// eslint-disable-next-line require-await
+		async ( { payload: { settingID, values } } ) => {
```

Comment:
Doesn't look like this needs to be `async` as it returns a Promise from `setItem`...